test/thread/TestKanoWorldRefresh.h

```diff
@@ -0,0 +1,106 @@
+/**
+ * \file kw_apis.h
+ *
+ * \copyright Copyright (C) 2019 Kano Computing Ltd.
+ *            License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+ *
+ * \brief Tests for the Mercury::KW - Kano World - functions
+ *
+ */
+
+#ifndef TEST_THREAD_TESTKANOWORLDREFRESH_H_
+#define TEST_THREAD_TESTKANOWORLDREFRESH_H_
+
+#include <gmock/gmock.h>
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <chrono>  // NOLINT
+#include <functional>
+#include <random>
+#include <string>
+#include <thread>  // NOLINT
```

Comment:
Interesting, linter complaining about `<thread>` also?

Reply:
Because Chromium has implementations of things which overlap with `<thread>`, `<future>`, `<mutex>` and `<condition_variable>` (i.e. the C++ standard threading libraries) these are forbidden by `cpplint` (a tool made by Google). See [here](https://chromium-cpp.appspot.com/) for more info.